### ISSUE TITLE
Apply ARIA compliance

### ISSUE THREAD

**Issue by clevend2** [clevend2/todo-vue#4]
**Is your feature request related to a problem? Please describe.**
For a note app, accessibility compliance would be a top priority since voice dictation and screen-readers should be fully enabled.

**Describe the solution you'd like**
All items on all screens to fully employ attributes and semantic structuring in keeping with ARIA standards.

**Describe alternatives you've considered**

**Additional context**
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA
